### DISCUSSION TITLE
NocoDB with Pterodactyl [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1921] [Discord Threads]
> 🙋‍♂️ **Mongo**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Could you please expand on this usecase  
#  
> 💁 **o1lab**  
#### we dont have much idea on Pterodactyl 🙂  
#  
> 🙋‍♂️ **Mongo**  
#### Yeah for sure! Pterodactyl is an open source server hosting service. They primarily target games for their servers, but I have been able to run node js and was wondering if anybody had any experience running NocoDB on it. If not is completely okay! It’s a bit of a long shot lol!  
#  
> 💁 **o1lab**  
#### If its the matter of running nodejs - nocodb is just 6 lines of nodejs 🙂    
#### see this  
#  
> 💁 **o1lab**  
#### https://github.com/nocodb/nocodb-seed/blob/main/index.js  
#  
> 🙋‍♂️ **Mongo**  
#### Ohhh!! This is nice! I’m definitely going to give this a try!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966229084902391808)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
